src/pz390.java

```diff
@@ -2788,10 +2798,16 @@ private void ins_lt_c0() {
 		 */
 		switch (opcode1) {
 		case 0x80: // 1530 "8000" "SSM" "S"
+             if (psw_problem_state == psw_problem_mode)
+                {set_psw_check(psw_pic_priv); // RPI 1622 DK Privileged operation exception
+                 } // in problem state issue privileged-operation exception, otherwise no-op until we implement this instruction
 			ins_setup_s();
 			break;
 		case 0x82: // 1540 "8200" "LPSW" "S"
 			psw_check = false;
+            if (psw_problem_state == psw_problem_mode)
+               {set_psw_check(psw_pic_priv); // RPI 1622 DK Privileged operation exception
+                } // in problem state issue privileged-operation exception
```

Comment:
Shift these 3 lines right one position. There are more instances below.

Reply:
layout is not really consistent. I agree it could be better. Current code indents to either column 14 or 15, varying throughout the entire source module. See e.g. SSM which has both indents :-( 
I'm in favor of running all source modules through a formatter to make it all consistent. But I'd prefer to do it as a separate change, not as part of this one...